tests/framework/installer/ceph_installer.go

```diff
@@ -492,12 +502,15 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, rook
 	if err != nil {
 		panic("failed to get kubectl client :" + err.Error())
 	}
+	// 0.8 is not able to handle the changed hostname until after this fix is backported.
+	// Then we can always change the hostname.
 	h := &CephInstaller{
-		Manifests:  NewCephManifests(rookVersion),
-		k8shelper:  k8shelp,
-		helmHelper: utils.NewHelmHelper(Env.Helm),
-		k8sVersion: version.String(),
-		T:          t,
+		Manifests:       NewCephManifests(rookVersion),
+		k8shelper:       k8shelp,
+		helmHelper:      utils.NewHelmHelper(Env.Helm),
+		k8sVersion:      version.String(),
+		changeHostnames: rookVersion != Version0_8 && k8shelp.VersionAtLeast("v1.11.0"),
```

Comment:
https://github.com/rook/rook/blob/9eab9470cf6fc205e1bbf298b8e253493d33f1d1/tests/framework/installer/installer.go#L35 is still `v0.8.1`, shouldn't it be `v0.8.2` (soon `v0.8.3`)?

Reply:
Yes, I noticed that as well. I want to update it as a separate PR and also consider how to get it to pick up the latest 0.8 release so we don't have to keep changing it.